[2024-03-20T05:02:49.547+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: example1.t4 manual__2024-03-20T05:02:28.045455+00:00 [queued]>
[2024-03-20T05:02:49.550+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: example1.t4 manual__2024-03-20T05:02:28.045455+00:00 [queued]>
[2024-03-20T05:02:49.550+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 2
[2024-03-20T05:02:49.555+0000] {taskinstance.py:2217} INFO - Executing <Task(BashOperator): t4> on 2024-03-20 05:02:28.045455+00:00
[2024-03-20T05:02:49.556+0000] {standard_task_runner.py:60} INFO - Started process 258 to run task
[2024-03-20T05:02:49.558+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'example1', 't4', 'manual__2024-03-20T05:02:28.045455+00:00', '--job-id', '22', '--raw', '--subdir', 'DAGS_FOLDER/example1.py', '--cfg-path', '/tmp/tmpkmkocj11']
[2024-03-20T05:02:49.559+0000] {standard_task_runner.py:88} INFO - Job 22: Subtask t4
[2024-03-20T05:02:49.565+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-03-20T05:02:49.576+0000] {task_command.py:423} INFO - Running <TaskInstance: example1.t4 manual__2024-03-20T05:02:28.045455+00:00 [running]> on host bf143d508a6e
[2024-03-20T05:02:49.608+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='***@example.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='example1' AIRFLOW_CTX_TASK_ID='t4' AIRFLOW_CTX_EXECUTION_DATE='2024-03-20T05:02:28.045455+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-03-20T05:02:28.045455+00:00'
[2024-03-20T05:02:49.610+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-03-20T05:02:49.610+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo task 4']
[2024-03-20T05:02:49.614+0000] {subprocess.py:86} INFO - Output:
[2024-03-20T05:02:49.615+0000] {subprocess.py:93} INFO - task 4
[2024-03-20T05:02:49.615+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-03-20T05:02:49.627+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=example1, task_id=t4, execution_date=20240320T050228, start_date=20240320T050249, end_date=20240320T050249
[2024-03-20T05:02:49.656+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-20T05:02:49.666+0000] {taskinstance.py:3312} INFO - 0 downstream tasks scheduled from follow-on schedule check
